Sources/TrustlySDK/TrustlyView.swift

```diff
@@ -482,7 +482,7 @@ public class TrustlyView : UIView, TrustlyProtocol, WKNavigationDelegate, WKScri
     func getEndpointUrl(function:String, establishData:[String:String]) -> String {
         var fn = function
         let env = establishData["env"]
-        let localUrl = establishData["localUrl"]
+        let envHost = establishData["envHost"]
```

Comment:
We can't change this names, because this will create incompatibility with the old versions of the SDK 3.X

Reply:
Yeah, we have a semantic impasse here. As this data is internal development-only, the idea was to improve the label since we have control over it.